PanKanapka/tools/csv2db.py

```diff
@@ -0,0 +1,86 @@
+from clients.models import User
+from clients.models import Company
+from sandwiches.models import Allergen
+from sandwiches.models import Ingredient
+from sandwiches.models import Sandwich
+import csv
+
+
+class DbManager:
+
+    allergens_csv_name = "doc/demo_data_in_csv/allergens.csv"
+    clients_csv_name = "doc/demo_data_in_csv/clients.csv"
+    ingredients_csv_name = "doc/demo_data_in_csv/ingredients.csv"
+    sandwiches_csv_name = "doc/demo_data_in_csv/sandwiches.csv"
+    companies_csv_name = "doc/demo_data_in_csv/companies.csv"
+
+
+    def import_allergens(self):
+        with open(self.allergens_csv_name) as allergens_csv_file:
+            csv_reader = csv.reader(allergens_csv_file, delimiter=',')
+            for row in csv_reader:
+                allergen = Allergen()
+                allergen.name = row[0]
+                allergen.save()
+
+    def import_ingredients(self):
+        with open(self.ingredients_csv_name) as ingredients_csv_file:
+            csv_reader = csv.reader(ingredients_csv_file, delimiter=',')
+            for row in csv_reader:
+                ingredient = Ingredient()
+                ingredient.name = row[0]
+                ingredient.group = None
+                ingredient.calories_per_portion = row[1]
+                ingredient.portion_size_grams = row[2]
+                ingredient.price = row[3]
+                ingredient.save()
+
+    def import_sandwiches(self):
+        with open(self.sandwiches_csv_name) as sandwiches_csv_file:
+            csv_reader = csv.reader(sandwiches_csv_file, delimiter=',')
+            for row in csv_reader:
```

Comment:
Idealnie byłoby ujednolicić kod. Czyli jak linii 52 do krotki odwołujesz się przez `name` i `address`, to w tym przypadku dobrze byłoby zrobić analogicznie (`name`, `price`, `accessible`, etc.).